src/App.elm

```diff
@@ -13,6 +13,7 @@ initialModel : Model
 initialModel =
     { currentContent = Pages.index
     , contributors = RemoteData.NotAsked
+    , searchPost = ""
```

Comment:
Whenever something is modelled with the string being empty I think that's better as a `Maybe`. I'd have this as a `Maybe String`, and when the value is `Nothing` we know not to bother searching.